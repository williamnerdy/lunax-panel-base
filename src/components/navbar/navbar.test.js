import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './navbar';

describe('<Navbar />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
