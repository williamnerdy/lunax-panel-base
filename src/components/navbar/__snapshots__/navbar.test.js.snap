// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <div
          style={
            Object {
              "color": "#fff",
              "float": "left",
            }
          }
        >
          <img
            alt="Logo"
            src="logo-small.png"
          />
          <span
            style={
              Object {
                "marginLeft": "15px",
              }
            }
          >
            Lunax Panel
          </span>
        </div>,
        <div
          style={
            Object {
              "float": "right",
            }
          }
        >
          <withRouter(UserOptions) />
        </div>,
      ],
      "className": "header",
      "style": Object {
        "background": "#2b2f3e",
        "color": "#fff",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Logo"
              src="logo-small.png"
            />,
            <span
              style={
                Object {
                  "marginLeft": "15px",
                }
              }
            >
              Lunax Panel
            </span>,
          ],
          "style": Object {
            "color": "#fff",
            "float": "left",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Logo",
              "src": "logo-small.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lunax Panel",
              "style": Object {
                "marginLeft": "15px",
              },
            },
            "ref": null,
            "rendered": "Lunax Panel",
            "type": "span",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <withRouter(UserOptions) />,
          "style": Object {
            "float": "right",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <div
            style={
              Object {
                "color": "#fff",
                "float": "left",
              }
            }
          >
            <img
              alt="Logo"
              src="logo-small.png"
            />
            <span
              style={
                Object {
                  "marginLeft": "15px",
                }
              }
            >
              Lunax Panel
            </span>
          </div>,
          <div
            style={
              Object {
                "float": "right",
              }
            }
          >
            <withRouter(UserOptions) />
          </div>,
        ],
        "className": "header",
        "style": Object {
          "background": "#2b2f3e",
          "color": "#fff",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Logo"
                src="logo-small.png"
              />,
              <span
                style={
                  Object {
                    "marginLeft": "15px",
                  }
                }
              >
                Lunax Panel
              </span>,
            ],
            "style": Object {
              "color": "#fff",
              "float": "left",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Logo",
                "src": "logo-small.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lunax Panel",
                "style": Object {
                  "marginLeft": "15px",
                },
              },
              "ref": null,
              "rendered": "Lunax Panel",
              "type": "span",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <withRouter(UserOptions) />,
            "style": Object {
              "float": "right",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
